Login/app/src/test/java/com/projetos/txoza/ValidadeUsers.kt

```diff
@@ -0,0 +1,25 @@
+package com.projetos.txoza
+
+import com.projetos.txoza.util.User
+import org.junit.Test
+import org.junit.Assert.*
+
+class ValidadeUsers {
+
+    val txoza = User("Txoza", "txoza@teste.com", "12345")
+
+    @Test
+    fun userIsValid() {
+        assertTrue(txoza.validateUSer("txoza@teste.com", "12345"))
+    }
+
+    @Test
+    fun userEmailInvalid() {
+        assertFalse(txoza.validateUSer("txoza@hotmail.com", "12345"))
+    }
+
+    @Test
+    fun userPasswordInvalid() {
+        assertFalse(txoza.validateUSer("txoza@teste.com", "1234"))
```

Comment:
Try create a structure for your tests as Given When Then

Ex:

// Given
val user = User()

// When
val result = user.isValid()

// Then
assert